src/ODriveUART.h

```diff
@@ -18,6 +18,9 @@ class ODriveUART {
      */
     ODriveUART(Stream& serial);
 
+    void selectAxis(int axis);
```

Comment:
this would have to be documented, especially that it can be ignored for all current-generation ODrives

Reply:
See above, could be mitigated with the overload